spec/controllers/subclassed_pages_controller_spec.rb

```diff
@@ -1,34 +1,34 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe SubclassedPagesController do
   render_views
 
-  describe 'on GET to /subclassed_pages/also_exists' do
-    before { get :show, :id => 'also_exists' }
+  describe "on GET to /subclassed_pages/also_exists" do
+    before { get :show, id: "also_exists" }
```

Comment:
Where the HTTP verbs now expect:
- A url / path instead of an action.
- Keyword arguments for `params`, `session`, and `headers`.